libra-node/src/lib.rs

```diff
@@ -384,7 +384,7 @@ pub fn setup_environment(node_config: &NodeConfig, logger: Option<Arc<Logger>>)
         Arc::clone(&db_rw.reader),
         chunk_executor,
         &node_config,
-        waypoint,
+        genesis_waypoint,
```

Comment:
I guess longer term ... if we want... this could be the real waypoint, but whatever

Reply:
yeah I was thinking if I want to use waypoint or just this, but since every time insert-waypoint will change both I don't think it matters.

LSR shouldn't be able to change the waypoint to a future value unless something weird happens (db wiped)